modules/travel_pay/app/controllers/travel_pay/claims_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module TravelPay
+  class ClaimsController < ApplicationController
+    def index
+      veis_token = client.request_veis_token
+
+      # Non-intuitive Ruby behavior: #split splits a string on space by default
+      vagov_token = request.headers['Authorization'].split[1]
+      btsss_token = client.request_btsss_token(veis_token, vagov_token)
+
+      begin
+        claims = client.get_claims(veis_token, btsss_token)
+      rescue Faraday::Error => e
+        raise common_exception(e)
```

Comment:
Is is probably not robust enough for the long term, but it's hard to predict what attributes we'll need to map from the upstream error response until we have docs.